app/javascript/packs/elm.js

```diff
@@ -20,6 +20,14 @@ document.addEventListener('DOMContentLoaded', () => {
     console.warn(params);
     console.warn(data);
 
+    const heatMapThresholdValues = data.heat ? {
+      threshold1: data.heat.lowest,
```

Comment:
what happens if those are undefined? (can they?)

Reply:
I think they are either there or the entire hash in `data.heat` is missing